plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectCsvRecordReader.java

```diff
@@ -95,8 +96,7 @@ public SelectObjectContentRequest buildSelectObjectRequest(Properties schema, St
 
     protected String getFieldDelimiter(Properties schema)
     {
-        // Use the field delimiter only if it is specified in the schema. If not, send it as null in the request to S3 Select.
-        // In this case, S3 Select defaults to using ',' as the field delimiter.
```

Comment:
This should be done right already in the `Enable Select pushdown on uncompressed files` commit